src/main/java/com/amadeus/Booking.java

```diff
@@ -0,0 +1,23 @@
+package com.amadeus;
+
+import com.amadeus.booking.FlightOrder;
+
+public class Booking {
+  private Amadeus client;
+
+  /**
+   * <p>
+   *   A namespaced client for the
+   *   <code>/v1/booking/flightOrder</code> endpoints.
```

Comment:
Is this class for /v1/booking/flightOrder endpoints or /v1/booking endpoints?

Reply:
The comment is related to `public FlightOrder flightOrder;`

Reply:
The path to call the post method is `amadeus.booking.flightOrder("eJzTd9f3NjIJdzUGAAp%2fAiY=").get();` and this is where this function is called and it's name should follow the path of the endpoint.